### DISCUSSION TITLE
about kafka runtime status in strimizi and parameters config

### DISCUSSION THREAD

**Discussion by ahu0605** [strimzi/strimzi-kafka-operator#6713] [General]
I want to know kafka in kubernetes how to I/O disk? local pv/pvc? how to graceful shutdown a cluster ?  how promotion the cluster performance for example  "num.recovery.threads.per.data.dir"? how to monitor leader elect? controller state?

**Comment by scholzj** [comment]
Please check the docs on https://strimzi.io which describe storage requirements, monitoring etc.
